examples/emberjs/todomvc/config/environment.js

```diff
@@ -1,10 +1,10 @@
-/* jshint node: true */
+/* eslint-env node */
 
 module.exports = function (environment) {
 	var ENV = {
 		modulePrefix: 'todomvc',
 		environment: environment,
-		baseURL: null,
```

Comment:
Hm, did I miscalculate and this must be `null` to make the `index.html` file have no leading `/`?

Reply:
Yes, I think that baseURL is deprecated. The important thing is that it works 🙂